src/rules/selector-no-empty/README.md

```diff
@@ -0,0 +1,47 @@
+# selector-no-empty
+
+Disallow empty selectors.
+
+```css
+    a { color: pink; }
+/** ↑ 
+ * This is selector */
+```
+
+Empty selectors (by themselves or within a selector list) invalidate a rule. This rule splits the selector list on, and any part of zero length or contains only whitespace is consider empty. Everything else isn't.
+
+## Options
+
+### `true`
+
+The following patterns are considered warnings:
+
+```css
+{ color: red} /* entirely empty selector */
```

Comment:
Please remove all the declarations from within the blocks for consistency with the other `selector-*` rule READMEs.

I think we can also remove the comments as I think the examples are pretty self-explanatory.